### ISSUE TITLE
测试代码是否不正确，只有RGB帧的输入

### ISSUE THREAD

**Issue by masac11** [Event-AHU/EventVOT_Benchmark#4]
感谢您的工作
我在运行测试代码的时候发现HDETrack/lib/test/evaluation/tracker.py文件内的event frame相关的输入都被注释掉了，貌似是RGB帧的输入，并没有事件的输入，取消相关注释后又遇到了其他问题，如果是的话希望您能重新上传下代码，谢谢🙏
<img width="1281" alt="image" src="https://github.com/Event-AHU/EventVOT_Benchmark/assets/28709431/a2863724-50b2-480a-8ec5-837d72dd4000">

**Comment by wsasdsda** [comment]
EventVOT数据集只有Event的数据，没有RGB。这里就是Event image的输入。

**Comment by masac11** [comment]
所以这里是因为非EventVOT数据集，导致读取到了RGB图片？

**Comment by wsasdsda** [comment]
@masac11 是的。如果要测VisEvent数据集的Event image，你可以在测试的时候将对应读取VisEvent测试集脚本的frames_path修改为读取Event image的路径
